Multisig/UI/Safe Management/Create Safe/CreateSafeFlow.swift

```diff
@@ -24,7 +25,9 @@ class CreateSafeFlow: UIFlow, ASAuthorizationControllerPresentationContextProvid
     private var relayingTask: URLSessionTask?
     let relayerService: SafeGelatoRelayService = App.shared.relayService
     private var appleWeb3AuthLogin: AppleWeb3AuthLogin!
-
+    private let uiModel = CreateSafeFormUIModel()
```

Comment:
it would be more consistent to retrieve this model also through factory. Also why is this reference to uiModel needed?

Reply:
It is needed because otherwise it'll be lost after `addOwnerAddress()` in line 160 in function `storeKeyAndCreateSafe()`

Reply:
Since we need it to be a property of the flow, I don't think it would be helpful to create it in a/the factory.